fedbadges/rules.py

```diff
@@ -50,6 +51,11 @@
 except ImportError as e:
     log.warn("Could not import email2fas: %r" % e)
 
+# Match OpenID agent strings, i.e. http://FAS.id.fedoraproject.org
+def openid2fas(openid, **config):
+    if re.match('^https?://[a-z][a-z0-9]+.id\.fedoraproject\.org', openid):
+        return re.search('^https?://([a-z][a-z0-9]+)',openid).group(1)
```

Comment:
Might be better to avoid repeating ourselves, here, with something like:
```
m = re.search('^https?://([a-z][a-z0-9]+)\.id\.fedoraproject\.org$', openid)
if m:
    return m.group(1)
```

Reply:
I like this a lot - it will be in the next update